pkg/apis/kops/validation/validation.go

```diff
@@ -1113,3 +1117,17 @@ func validateNodeLocalDNS(spec *kops.ClusterSpec, fldpath *field.Path) field.Err
 
 	return allErrs
 }
+
+func validateClusterAutoscaler(cluster *kops.Cluster, spec *kops.ClusterAutoscalerConfig, fldPath *field.Path) (allErrs field.ErrorList) {
+	if !cluster.IsKubernetesGTE("1.15") || cluster.IsKubernetesGTE("1.20") {
+		allErrs = append(allErrs, field.Forbidden(fldPath, "Cluster autoscaler requires kubernetesVersion between 1.11 and 1.19"))
```

Comment:
I don't think there's any other validation that puts an upper bound on the Kubernetes version for a reason other than removal of support. Cilium, for example, allows the highest known Cilium version be used for arbitrary newer Kubernetes versions. So I don't think we should cap this either.

Reply:
The reason for this one was "remembering to add support for newer autoscaler". Original suggestion was to panic in the template function. Third option is to use the latest tag when we have an unknown/unsupported K8s version.

I suspect we'll bump the upper bound long before the first kops alpha that supports the new k8s version.

Reply:
Could 1.13 and 1.14 also be supported, this would cover all supported versions? Also validation lower limit is for 1.15 and message says 1.11.
I also think that shouldn't be an upper limit and just use the one for the previous version.

Reply:
I just added the versions that had a release the last 6 months or so. Can add more, but I am not 100% sure the manifests will work that far back. 

My initial PR used latest tag for higher versions. But using latest release works for me too. Probably doesn't break anything.

Reply:
I am ok with latest as the default tag, as long as we add the rest of the images. Probably the tag should be changed to full image name to allow older image names.
The Helm chart seems to work for Kubernetes 1.8+ with the same manifest, so should be ok.

Reply:
Now that you added the image override, I don't mind if we have the upper limit for this as long as it is ignored when the image is overridden. Your call.